src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_utils.hpp

```diff
@@ -68,10 +76,8 @@ size_t compute_inner_k_block(const ov::element::Type& precision);
 template <
     typename T,
     typename = typename std::enable_if<(std::is_same<T, size_t>::value || std::is_same<T, int64_t>::value), bool>::type>
-T compute_LDB(T n_block, const ov::element::Type& precision) {
-    return snippets::utils::is_dynamic_value<T>(n_block)
-               ? n_block
-               : std::max(n_block, static_cast<T>(compute_inner_n_block(precision)));
+inline T compute_LDB(T n_block, const ov::element::Type& precision) {
+    return compute_repacked_n_dim(n_block, precision);
```

Comment:
Why do we need 2 different functions that do the same thing?
Should we replace all `compute_LDB` calls with `compute_repacked_n_dim` then?

Reply:
I left the both them just to split logic into `get_LDB` and `get_repacked_N` by sense.
But the implementation is the same.

Anyway, I replaced `compite_LDB` with `compute_repacked_n_dim` in https://github.com/openvinotoolkit/openvino/pull/28664/commits/15dadd5ac72be0f218f26ceb5809aec52f047413